src/main.cpp

```diff
@@ -100,77 +100,77 @@ base::Vector parsePlayerPosition(const std::string& playerPosString) {
 
 
 int main(int argc, char** argv) {
-  showBanner();
+  // showBanner();
```

Comment:
Do things still work if you leave in the call to `showBanner`? It's not really necessary, but it should not depend on `program_options`, so I'm curious if there was some other problem with it